packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -62,8 +62,10 @@ export const GRID_PAGESIZE_CHANGED = 'pageSizeChange';
 export const GRID_ROWS_SCROLL = 'scrolling:rows';
 export const GRID_ROWS_SCROLL_END = 'scroll:rowEnd';
 
-export const GRID_COL_RESIZE_START = 'colResizing:start';
-export const GRID_COL_RESIZE_STOP = 'colResizing:stop';
+export const GRID_COLUMN_RESIZE = 'columnResize';
+export const GRID_COLUMN_RESIZE_MOUSE_DOWN = 'columnResize:MouseDown';
```

Comment:
For consistency?

```suggestion
export const GRID_COLUMN_RESIZE_MOUSE_DOWN = 'columnResize:mouseDown';
```

Reply:
That's wrong as an event name! You don't do resize on mouse down!
You do a ColumnHeaderMouseDown

Reply:
I think this is just a annotation to describe what the `mouseDown` is related to.

Reply:
Updated